src/sentry/data_export/tasks.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import absolute_import
```

Comment:
This whole file is diffed but only for a few changes:
- New import locations
- process functions no longer return file_names (this is handled on the model itself now)
- some `six.text_type` wrappers on error messages
- `snuba_error_handler` moved to the adjacent `utils.py`
- sends the error message as a string to the failure email function

Reply:
The reason it's diffed is because the original `tasks/data_export.py` file has been deleted, and this is the copy/pasted `data_export/tasks.py` file. The same queue is in place for both, so the config should still apply from here https://github.com/getsentry/ops/commit/5036032e0dcd5c90623bff8ea6d3adbf375ea22e